_inc/client/components/welcome-new-plan/videopress-prompt.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { imagePath } from 'constants/urls';
+import InlineModuleToggle from 'components/module-settings/inline-module-toggle';
+
+class VideoPressPrompt extends React.Component {
+	render() {
+		return <div>
+			<img src={ imagePath + 'wordads.svg' } className="jp-welcome__svg" alt={ __( 'VideoPress' ) } />
```

Comment:
Since this icon is the same one we're using for the social and ads section, does it make sense to group the settings together under a single instance of the icon?